internal/helm/release/manager.go

```diff
@@ -370,5 +370,8 @@ func (m manager) UninstallRelease(ctx context.Context, opts ...UninstallOption)
 		}
 	}
 	uninstallResponse, err := uninstall.Run(m.releaseName)
+	if uninstallResponse == nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	if err != nil {
		return nil, err
	}
```

Reply:
hi @estroz I think originally I have the same thing as what you are suggesting. But then I look at the current code for install and upgrade:
https://github.com/operator-framework/operator-sdk/blob/v1.2.0/internal/helm/release/manager.go#L171-L179
and
https://github.com/operator-framework/operator-sdk/blob/v1.2.0/internal/helm/release/manager.go#L212-L219

So I think its possible that `err` is not nil and `uninstallResponse` is not nil as well.
That's why I am returning both and let the caller decide what to do in that case.

Reply:
It looks like the only caller does indeed use both even if `err != nil`, so your original code is fine. However the caller is also at risk of a [panic](https://github.com/operator-framework/operator-sdk/blob/v1.2.0/internal/helm/controller/reconcile.go#L119-L125) even if this fix is applied.

Reply:
I see. Thanks for the nice check. What do you suggest we do in this case then? Should I enhance my PR some more with the fixes to the reconcile code?

Reply:
In the caller:
```go
			log.Info("Uninstalled release")
			if log.V(0).Enabled() && uninstalledRelease != nil {
				fmt.Println(diff.Generate(uninstalledRelease.Manifest, ""))
			}
```

Reply:
Done as suggested. Thanks again.